OlxProject/pages/log_register_page.rb

```diff
@@ -0,0 +1,120 @@
+require_relative '../tools/all_requires'
+
+class LogRegisterPage
+
+  include TopPart
+
+  protected def login_tab
+    driver.find_element(id: 'login_tab')
+  end
+
+  protected def log_user_email_field
+    driver.find_element(id: 'userEmail')
+  end
+
+  protected def log_user_pass_field
+    driver.find_element(id: 'userPass')
+  end
+
+  protected def log_user_button
+    driver.find_element(id: 'se_userLogin')
+  end
+
+  protected def log_fb_button
+    driver.find_element(id: 'fblogin')
+  end
+
+  protected def captha                              # TODO
+    driver.find_element(id: 'recaptcha-anchor')
+  end
+  # -----------------------------------------------------------------------
+  protected def register_tab
+    driver.find_element(id: 'register_tab')
+  end
+
+  protected def reg_user_pass
+    driver.find_element(id: 'userPassRegister')
+  end
+
+  protected def reg_user_email_field
+    driver.find_element(id: 'userEmailPhoneRegister')
+  end
+
+  protected def reg_accept_checkbox
+    driver.find_element(xpath: "//input[contains(@id,'checkbox_accept-terms')]/../../.././label[contains(@class,'icon f_checkbox inlblk vtop')]")
```

Comment:
easier selector? can we create additional element like checkbox_container and find element from it's location with class selector?